cluster-autoscaler/cloudprovider/azure/azure_vms_pool.go

```diff
@@ -154,11 +154,11 @@ func (vmPool *VMPool) IncreaseSize(delta int) error {
 	if len(versionedAP.Properties.VirtualMachinesProfile.Scale.Manual) > 0 {
 		requestBody = buildRequestBodyForScaleUp(versionedAP, count, vmPool.sku)
 
-	} else { // AKS-managed CAS will use custom header for setting the target count
-		header := make(http.Header)
-		header.Set("Target-Count", fmt.Sprintf("%d", count))
-		updateCtx = policy.WithHTTPHeader(updateCtx, header)
 	}
+	header := make(http.Header)
```

Comment:
With this change we appear to be setting HTTP headers that include the Target-Count key/value pair, and hoisting that into the context object that we send to the Azure API, for *both* self-hosted _and_ AKS-managed. Whereas now we're only doing this in the AKS-managed case.

Why are we making this change to include this behavior for both cases going forward?

Reply:
It is because Hobo systempool will only have manual scale profile, a simple if-else check is no longer sufficient to distinguish between self-hosted and managed CAS.

I just realized I could add an explicit check for HOBO to make the logic clearer, so I’ve updated the code accordingly. Let me know if that makes sense. Thanks!

Reply:
+1, thanks!